apps/meteor/client/components/MarkdownText.tsx

```diff
@@ -14,6 +15,27 @@ type MarkdownTextParams = {
 	withTruncatedText: boolean;
 } & ComponentProps<typeof Box>;
 
+const getBaseURI = (): string => {
```

Comment:
`question` Why is this function duplicating the IIFE at `apps/meteor/client/lib/baseURI.ts`?

Reply:
fair question :) where do you suggest I should export it from?